handlers.go

```diff
@@ -120,7 +128,7 @@ func checkLoginStatus(w http.ResponseWriter, r *http.Request) *internal.User {
 		http.Error(w, "Invalid login session.", http.StatusBadRequest)
 		return nil
 	} else if id < 1 {
-		log.Println("Invalid user id -", err)
+		log.Println("Invalid user id:", id)
```

Comment:
For consistency, shouldn't this be log.Error?

Reply:
Log.Error doesn't exist, it's only used for tests as "t.Error" indicates "t.Log() and t.Fail() this test.

Reply:
Never mind my apologies for the oversight.